server/src/messageHandler.mjs

```diff
@@ -108,16 +109,16 @@ async function handleMessage(message, userId, ws) {
   }
 
   // Fire pre trigger if there is one for this method  
-  if ( oMsg.method in methodTriggers ) {
-    if ( 'pre' in methodTriggers[oMsg.method] ) {
+  if (oMsg.method in methodTriggers) {
```

Comment:
Please don't go out of your way to change the style of existing code. It's fine as-is, but the code base uses spaces within parens in conditionals and I don't see a good reason to change. Thanks.